src/components/views/dialogs/ExportDialog.tsx

```diff
@@ -174,12 +176,16 @@ const ExportDialog: React.FC<IProps> = ({ room, onFinished }) => {
             {
                 key: "number",
                 test: ({ value }) => {
+                    const config = SdkConfig.get();
                     const parsedSize = parseInt(value!, 10);
-                    return validateNumberInRange(1, 2000)(parsedSize);
+                    const max = config.max_export_size_mb;
+                    if (typeof max === "undefined") throw new Error("max export size undefined in settings");
```

Comment:
This seems un-idiomatic, at least. I feel that this shouldn't be undefined ever given the default, but maybe I've got that wrong?